endpoints/get_transactions.py

```diff
@@ -60,13 +70,22 @@ class TxSearch(BaseModel):
     transactionIds: List[str]
 
 
+class PreviousOutpointLookupMode(str, Enum):
+    no = "no"
+    light = "light"
+    full = "full"
+
+
 @app.get("/transactions/{transactionId}",
          response_model=TxModel,
          tags=["Kaspa transactions"],
          response_model_exclude_unset=True)
 async def get_transaction(transactionId: str = Path(regex="[a-f0-9]{64}"),
                           inputs: bool = True,
-                          outputs: bool = True):
+                          outputs: bool = True,
+                          resolve_previous_outpoints: PreviousOutpointLookupMode =
+                          Query(default="no",
```

Comment:
Same comment here about using the enum to assert this value is valid. And apply it to all other instances in this file

Reply:
Yep, fixed this!